test/test_05_01.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime, timedelta
+
+from hm_1.tasks.task_05_01 import Homework, Student, Teacher
+
+
+def test_class_homework_atributes_saved():
+    attributes = (
+        "Learn function",
+        timedelta(days=5),
+        datetime(2019, 1, 12, 15, 44, 35),
+    )
+    assert list(Homework(*attributes).__dict__.values()) == list(attributes)
```

Comment:
It is better to make a separate assert statement for each attribute.  It will be more simple.